### DISCUSSION TITLE
[Update request] Update noto-fonts to the latest version (2.010/2.009)

### DISCUSSION THREAD

**Discussion by jopejoe1** [ryanoasis/nerd-fonts#1071] [Ideas]
The noto-fonts currently included is on version v2017-10-24-phase3-second-cleanup which was released in 2017 where the current version of noto-fonts is 2.010 which was released in 2023. The latest releases can be found at https://github.com/notofonts/latin-greek-cyrillic/releases.

**Comment by jopejoe1** [comment]
Alternately can also be downloaded from https://github.com/notofonts/notofonts.github.io/tree/main/fonts

**Comment by Finii** [comment]
:thinking: Why is that not on the [list](https://github.com/ryanoasis/nerd-fonts/discussions/962#discussioncomment-3894678)

All source fonts will be updated on `v3.0.0`
